test/integration/test_ncbi.py

```diff
@@ -20,7 +20,7 @@
 log = logging.getLogger(__name__)
 
 
-skip_test = True
+skip_test = False
```

Comment:
These tests were mostly disabled because instability in the NCBI API due to load frequently causes the tests to fail, even with the retries in `genbank.py`. Enabling them was mostly to illustrate the tempdir issue, so we can probably switch them off again.